### DISCUSSION TITLE
What is the default rustdesk server hostname?

### DISCUSSION THREAD

**Discussion by bilogic** [rustdesk/rustdesk#7268] [Q&A]
Hi,

1. Under Settings, Network, we can specify an ID and Relay server.
2. What are the default values that RustDesk uses if we don't specify anything?

**Comment by xlionjuan** [comment]
https://github.com/rustdesk/uptime/commit/597397e145e7030a2b4fe28278c89fe4d11795c5

**Comment by C-Duv** [comment]
According to `summary.json` of https://github.com/rustdesk/uptime the *Public Rendezvous Server*'s FQDN is `rs-ny.rustdesk.com`.

But [it seems there are multiple *Public Relay Servers*](https://github.com/rustdesk/uptime/blob/master/history/summary.json) and their URL are in GitHub Actions variables:

* RustDesk Public Relay Server - EU1: `$HZ_SERVER`
* RustDesk Public Relay Server - EU2: `$EU2_SERVER`
* RustDesk Public Relay Server - East: `$EAST2`
* RustDesk Public Relay Server - West: `$WEST`
* RustDesk Public Relay Server - Tok: `$TOK`

How does the client decide which relay server to use? Or maybe it doesn't and don't decide by itself and only use a relay server if user configured it to do so?

**Reply by xlionjuan** [reply]
GeoIP or Ping I guess
But according to the documentation of RustDesk pro, it has the ability to import geoIP to hbbs, so hbbs can decide which to go

**Reply by bilogic** [reply]
Rendezvous = `ID` or `Relay` server?
hbbs = `ID` or `Relay` server?

1. My end goal is to always point my rustdesk to a custom hostname
2. During good times, custom hostname will point to rustdesk's default
3. During outages, all I have to do is update my DNS to point to my own server, no change to all my clients
4. I have both hbbr and hbbs running and tested working, but no idea which is `ID`, which is `Relay` and then there is `Rendezvous`...
5. We should stick to a consistent naming

**Reply by xlionjuan** [reply]
Rendezvous=ID=heartbeat=hbbs

Relay is relay, no other name
